composer.json

```diff
@@ -9,15 +9,19 @@
     }
   ],
   "require": {
-    "php": ">=7.1"
+    "php": ">=7.1",
+    "fond-of-spryker/allowed-product-quantity-cart-connector": "dev-master",
+    "spryker/glossary-storage": "^1.0.0",
```

Comment:
@SpiderDan98 did you consider all possible versions?

Reply:
@daniel-rose No, but everything's okay here.